x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/common/components/event_log/event_log_data_grid.tsx

```diff
@@ -142,7 +142,7 @@ const columnsWithToolTipMap: Record<string, Record<string, string>> = {
 
 export const ColumnHeaderWithToolTip = ({ id }: { id: string }) => {
   return (
-    <EuiFlexGroup gutterSize="xs" alignItems="center">
+    <EuiFlexGroup gutterSize="xs" alignItems="center" style={{ display: 'inline-flex' }}>
       <EuiFlexItem>{columnsWithToolTipMap[id].display}</EuiFlexItem>
       <EuiFlexItem>
         <EuiIconTip
```

Comment:
When I saw this issue, I thought we could just move the `EuiIcon` inside the same FlexItem as the text itself so that they always stayed together 😅 Did you try that?